### PR TITLE
Rebroadcasting requests is enabled and configurable

### PR DESCRIPTION
**Pull request by Juliusan**
Rebroadcasting of off-ledger requests is now once again enabled. To avoid it causing problems, the time period for it can be configured via `chains.broadcastInterval` parameter. It is set to 1 hour by default.

### CODE REVIEW

File: tools/cluster/templates/waspconfig.go [hunk 1]
```diff
@@ -87,7 +87,7 @@ var WaspConfig = `
   },
   "chains": {
     "broadcastUpToNPeers": 2,
-    "broadcastInterval": "5s",
+    "broadcastInterval": "1s",
```
Comment (jorgemmsilva) [inline tools/cluster/templates/waspconfig.go]: won't this make the spammy tests even more exhausting to run?

File: components/chains/params.go [hunk 2]
```diff
@@ -8,7 +8,7 @@ import (
 
 type ParametersChains struct {
 	BroadcastUpToNPeers              int           `default:"2" usage:"number of peers an offledger request is broadcasted to"`
-	BroadcastInterval                time.Duration `default:"5s" usage:"time between re-broadcast of offledger requests"`
+	BroadcastInterval                time.Duration `default:"1h" usage:"time between re-broadcast of offledger requests"`
```
Comment (jorgemmsilva) [inline components/chains/params.go]: should be possible to disable this feature altogether IMO, maybe accepting 0 as disabled

**Review by jorgemmsilva — APPROVED** [review]

File: components/chains/params.go (hunk 2 above)
Comment (Juliusan) [reply components/chains/params.go]: Karolis offered it. I went with easier solution :-) I'll implement the zero value option then. I have to check how `0` durations compare to each other :-)

File: tools/cluster/templates/waspconfig.go (hunk 1 above)
Comment (Juliusan) [reply tools/cluster/templates/waspconfig.go]: Probably. I was happy regular cluster tests ran without any problems. I'll check the spam tests than.

**Review by kape1395 — APPROVED** [review]
I agree with the other comments.

File: tools/cluster/templates/waspconfig.go (hunk 1 above)
Comment (jorgemmsilva) [reply tools/cluster/templates/waspconfig.go]: just set it to a high value, no? we don't even test this stuff in the cluster tests afaik.

**Pull request merged**